src/views/feed/ActivityFeed/components/GroupMessageItem.vue

```diff
@@ -7,10 +7,7 @@
       :avatar="avatar"
     >
       <template>
-          {{ notification.target2_name + ' ' }} {{ isInvite ? $t('activity-feed:invite') : $t('activity-feed:broadcast') + ' ' }}
-          <router-link :to="`/groups/${nid}`">
-              {{ title }}
-          </router-link>
+          {{ notification.target2_name + ' ' }} {{ isInvite ? $t('activity-feed:invite') : $t('activity-feed:broadcast') + ' ' }}{{ title }}
```

Comment:
I just realized - for group invites, I think the NID should be included, which is important for you to be able to accept the invitation!

Reply:
JK, the invite link is in the group body